### ISSUE TITLE
Uncomment kube-dns metrics port

### ISSUE THREAD

**Issue by willvrny** [apprenda/kismatic#880]
**Is this a BUG REPORT or FEATURE REQUEST?**:

FEATURE REQUEST

**What happened**:

When curling the metrics endpoint for the kubedns container (http://<pod_ip>:10055/metrics) it errored.

**What you expected to happen**:

I expected to be able to get metrics for kubedns container on port 10055.

**How to reproduce it (as minimally and precisely as possible)**:

Run a busybox-curl container, describe a kube-dns pod to get it's IP address and then curl that IP address with port `10055` and a `/metrics` endpoint.

**Anything else we need to know?**:

To fix this we need to simply uncomment the env variable in the template: `kismatic/ansible/roles/kube-dns/templates/kubernetes-dns.yaml`

```
        # env:
        # - name: PROMETHEUS_PORT
        #   value: "10055"
```

**Comment by dkoshkin** [comment]
@willvrny were you able to test if that is actually the correct port that responds with metrics?

**Comment by willvrny** [comment]
@dkoshkin I edited the deployment on a running cluster and was able to receive metrics. 10055 is for the kubedns metrics and 10054 is for dnsmaq metrics.

**Comment by dkoshkin** [comment]
Fixed in #881, thanks @willvrny!
